tests/core/test_core.py

```diff
@@ -444,14 +445,13 @@ def test_pulse_model_and_ljh2off(self, tmp_path_factory):
 
         should_be_identity = np.matmul(pulse_model.projectors, pulse_model.basis)
         wrongness = np.abs(should_be_identity - np.identity(n_basis))
-        # ideally we could set this lower, like 1e-9, but the linear algebra needs more work
+        print(f"Wrongness matrix (abs-max is {np.amax(wrongness)})")
         print(wrongness)
-        print(np.amax(wrongness))
-        assert np.amax(wrongness) < 0.16
+        assert np.amax(wrongness) < 1e-9
```

Comment:
We have achieved the ideal!??

Reply:
Apparently, yes. I don't know why this was broken before, but it sure isn't broken now.